db/schema.rb

```diff
@@ -39,8 +39,6 @@
     t.integer "likes_counter", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "user_id", null: false
-    t.index ["user_id"], name: "index_posts_on_user_id"
```

Comment:
- [ ] Great job so far @ritikarawat220 There are just a few changes you need to make and you'll be on your way to the next milestone. :wink: 
- [ ] I noticed that when the previous reviewer mentioned that you have both `author_id` and `user_id` in your posts table so you needed to delete the `user_id` from your table, you only deleted it in the schema file. However, this action didn't remove the `user_id` from the database in the `posts table`. _You can test this by deleting the current schema file and rerunning the migrations. You'll notice that the user_id will be added once again._ :+1: 
- [ ] As a result, when I tried to create a post, I also got the same error message the previous reviewer also got. :worried: 
- [ ] I will suggest that just as the previous reviewer mentioned kindly consider removing `user_id`, added by the last migration file. You can delete that file completely. After that, you can drop the existing database and recreate it. That way you can rerun all the migrations thereby ensuring that your database only contains the `author_id` under the posts table :wink: 
- [ ] After this please test the comments and likes features and ensure that they work as expected :wink: 

- Error message
![image](https://github.com/ritikarawat220/Blog-App/assets/102699958/736ef12f-cdef-420a-b813-b8dcd1039b67)